commit 6e16d1e3b8f55106660aa2de4133f55acb06e06c (refs/changes/26/1126/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-14T16:22:27-08:00 (2 years, 10 months ago)
    
    IMGAPI-606 imgapi-cli commit 2aef5b0 broke feature that 'identity' can be a *path to a key file*

diff --git a/CHANGES.md b/CHANGES.md
index e4489d4..fca1665 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,12 @@
 
 ## not yet released
 
+## 2.2.4
+
+- IMGAPI-606 Fix '--identity' (and the equiv `*_IMGADM_IDENTITY` env var) to
+  once again support taking a path to an SSH private key. This was broken
+  in version 2.0.0.
+
 ## 2.2.3
 
 - TOOLS-1636 Fix updates-imgadm et al to flush stdout on exit.
diff --git a/package.json b/package.json
index cb972e4..0373f43 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi-cli",
   "description": "a CLI for an Triton's IMGAPI (https://images.joyent.com/docs/)",
-  "version": "2.2.3",
+  "version": "2.2.4",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -11,7 +11,7 @@
     "mkdirp": "0.5.1",
     "nopt": "2.0.0",
     "progbar": "1.1.1",
-    "sdc-clients": "10.0.3",
+    "sdc-clients": "10.0.4",
     "strsplit": "1.0.0",
     "tabula": "1.8.0",
     "vasync": "1.6.4",
